modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -35,108 +36,144 @@ After the new adapter is ready, the old pod is terminated and all requests are r
 
-image::service-mesh-adapter-binding.png[]
+[source,text,linenums]
+----
+NAMESPACE      NAME        ADAPTER                         API ID  SERVICE NAME    STATUS
+nto-payments   customers   anypoint-service-mesh-adapter   1234    customers       Ready
+----
 
 === CLI Help for Adapter Commands
 
 The command line interface (CLI) tool for Service Mesh provides help for various commands. You can issue help commands at any time after you complete installing Service Mesh. 
 
 To use the help, type the following command:
-`./service mesh help`
++
+`./service-mesh help`
 
 image::service-mesh-cli-adapter-help.png[CLI Help for Adapter Commands]
 
 == Provision Service Mesh Adapter
 
-Before	you can start managing your API using Service Mesh, you must prepare your namespace by provisioning a Service Mesh Adapter. Provisioning prepares the namespace for the Service Mesh configuration.
+Before	you can start managing your API using Service Mesh, you must prepare your namespace by provisioning a Service Mesh Adapter. Provisioning prepares the namespace for the Service Mesh configuration. After you provision the adapter, you must redeploy your existing application to ensure that the Envoy sidecar is injected with each POD.
 
 To provision the Anypoint Service Mesh adapter:
  
 . Determine the adapter size based on the plan.
-For more information, see...
 . From the Service Mesh installer download location in the Command window, type the following command to create the adapter:
 +
 [source,text,linenums]
 ----
-./service-mesh adapter create \--name=anypoint-service-mesh-adapter \--namespace=marketing \
-
---size=medium \--clientId=<replace_with_clientId> \
-
---clientSecret=<replace_with_clientSecret> \
-
---platformUri=https://anypoint.mulesoft.com
+./service-mesh adapter create 
+--name=<adapter name>
+--namespace=<adapter namespace>
+--size=<adapter plan size>
+--clientId=<clientId of the application> 
+--clientSecret=<clientSecret of the application> 
+--platformUri=<URL of Anypoint Platform>
 ----
-+	
-Where:
 +
 [%header%autowidth.spread]
 |===
 | Parameter Name | Description | Required/Optional | Default Value
-| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. For the Beta exercise, use “anypoint-service-mesh-adapter” for this parameter. | Required | N/A
-| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. For the Beta exercise, use “marketing” for this parameter | Optional | default
+| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. | Required | N/A
+| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. | Optional | default
 | size | The size of the adapter based on the number of CPUs and memory to use | Optional | Medium
```

Comment:
Size "Medium" has to be lower case "medim"

Reply:
Done!